gems/policy-parser/Dockerfile.test

```diff
@@ -7,6 +7,8 @@ COPY Gemfile Gemfile
 COPY conjur-policy-parser.gemspec conjur-policy-parser.gemspec
 COPY lib/conjur-policy-parser-version.rb lib/conjur-policy-parser-version.rb
 
+RUN apt-get update && apt-get install -y libffi-dev build-essential libyaml-dev
```

Comment:
Hi @hughsaunders, did you considered using the cyberark/ubuntu-ruby-builder:latest as a base image it comes with everything needed for installing the native extensions. Additionally it has FIPS mode disabled which makes bundler install faster (since bundler uses md5 based optimization while downloading gems).
To enable the FIPS mode, after installing all the gem, you could simple add:
```
ENV OPENSSL_CONF=/usr/lib/ssl/openssl.cnf
```

Reply:
Hey @tarnowsc, I didn't know of this image, thanks will try it 👍

Reply:
Pushed a commit with this suggestion